frontend/src/app/app.component.html

```diff
@@ -1,533 +1,4 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
```

Comment:
seems like your functionality doesn't affect this code, so why you deleted this component content?

Reply:
done